plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -196,10 +196,11 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "By default, the texture cache handles partial invalidations. Unfortunately it is very costly to compute CPU wise."
 				"\n\nThis hack replaces the partial invalidation with a complete deletion of the texture to reduce the CPU load.\n\nIt helps snowblind engine games.";
 		case IDC_LARGE_FB:
-			return "Allocate a large framebuffer to be compliant with GS memory (Prevents FMV flickering).\n"
-				"It increases GPU/memory requirements.\n\n"
-				"Note: It should be disabled for Armored Core, Destroy All Humans, and Gran Turismo.\n"
-				"It will amplify RAM/VRAM spikes. Other games might be affected as well.";
+			return "Reserves a bigger sized framebuffer for textures to prevent FMV flickers.\n"
+				"Increases GPU/memory requirements.\n"
+				"Enabling this can amplify stuttering due to low RAM/VRAM.\n\n"
+				"Note: It should be disabled (for Armored Core, Destroy All Humans, Gran Turismo and others).\n"
+				"This option does not improve the graphics or the FPS.";
```

Comment:
Last line seems a bit useless.

Reply:
I can understand why he put it there though :P

Reply:
It's the balance i made where it implied it's a bad option, i made this extra line to make it obvious to people (even my former self) that it should only be used for FMV flickering.